ner_telhub/widgets/styled_widgets.py

```diff
@@ -42,3 +44,34 @@ def changeStyle(self, style: Styles):
         self.style = style
         self.setStyleSheet(style.value)
 
+"""
+This top class is really just a place holder for it someone wants 
+to add any GUI thing to a toolbar
+"""
+class Ner_toolbar(QToolBar):
```

Comment:
Rename to NERToolbar. We should add a simple init method that class the super constructor (for QToolBar), and then a few methods to allow for adding buttons. I would imagine these methods would be something like:
- `def addButtonLeft(image_path or text)`
- `def addButtonRight(image_path or text)`
Also look into using toolbar buttons: https://doc.qt.io/qt-6.2/qtoolbutton.html